src/frontend/src/apis/work-packages.api.ts

```diff
@@ -53,7 +53,10 @@ export const getSingleWorkPackage = (wbsNum: WbsNumber) => {
  *
  * @param payload Payload containing all the necessary data to create a work package.
  */
-export const createSingleWorkPackage = (payload: CreateWorkPackageApiInputs) => {
+export const createSingleWorkPackage = (payload: CreateWorkPackageApiInputs | void) => {
+  if (payload === undefined) {
```

Comment:
Instead of checking void here, could you check void where this function is called in work-packages.hooks.ts